src/main/kotlin/com/chattriggers/ctjs/minecraft/libs/renderer/Renderer.kt

```diff
@@ -457,15 +457,15 @@ object Renderer {
 
     @JvmOverloads
     @JvmStatic
-    fun drawString(text: String, x: Float, y: Float, shadow: Boolean = false) {
+    fun drawString(text: String, x: Float, y: Float, color: Int = Renderer.WHITE, shadow: Boolean = false) {
```

Comment:
The default should be `colorized ?: Renderer.WHITE`, same applies for the other method. Also missing a breaking TODO here

Reply:
Alright, I didn't know you could do notation like that. I do believe `color` is a more appropriate name.

What do you mean with "missing a breaking TODO"